From 11d807379fcb574b86ab54b80101be45c104a774 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@serpentos.com>
Date: Wed, 7 Aug 2024 08:58:09 +0000
Subject: [PATCH] justfile: Use relative links for DESTDIR style installs

This resolves an issue with Serpent OS when the installed plugin
links point back to `/mason/install/usr/bin/pop-launcher`

Signed-off-by: Ikey Doherty <ikey@serpentos.com>
---
 justfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/justfile b/justfile
index 58d9ff8..06793e9 100644
--- a/justfile
+++ b/justfile
@@ -70,7 +70,7 @@ install-plugins:
         dest={{plugin-dir}}${plugin}
         mkdir -p ${dest}
         install -Dm0644 plugins/src/${plugin}/*.ron ${dest}
-        ln -sf {{bin-path}} {{plugin-dir}}${plugin}/$(echo ${plugin} | sed 's/_/-/')
+        ln -srf {{bin-path}} {{plugin-dir}}${plugin}/$(echo ${plugin} | sed 's/_/-/')
     done
 
 # Install pop-launcher scripts
-- 
2.46.0

